sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_responses.hpp

```diff
@@ -82,12 +82,12 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares { names
     /**
      * @brief Creation time for the file/directory..
      */
-    Azure::Core::Nullable<std::string> CreationTime;
+    Azure::Core::Nullable<Core::DateTime> CreateOn;
```

Comment:
`CreateOn` or `CreatedOn`?

Reply:
CreatedOn and LastModifiedOn

Reply:
Nah, we talked about it in the review meeting. It's `CreatedOn` and `LastModified`.